pkg/utils/digits.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright The CloudNativePG Contributors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils
+
+import "fmt"
+
+// CountDigits counts number of digits in an integer.
+func CountDigits(number int) int {
```

Comment:
Now doesn't make sense to have this function in one file and one file to test this function, since it's an inliner it can be used directly when you need the value

Reply:
good point